### ISSUE TITLE
Simpler event types (no nesting)

### ISSUE THREAD

**Issue by vivienlacourba** [w3c/w3c-website#34]
Discussing Event types with @koalie, @jean-gui and @alexandralacourba we concluded that nesting of event types was not desired.

Therefore we request that:
- "AC Meeting" and "TPAC meeting" would become level 1 event types (independent from the current "Global Meetings")
- Slug for "AC Meeting" should be "ac"
- Slug for "TPAC Meeting" should be "tpac"
- It should not be allowed to have sub event types (we stay with a simple list with no children)
- "TPAC" should be added to the "News and events" menu but not "AC" (as AC meetings are non public events, and would be better advertised from the Member homepage).

Those changes have been reflected in the sitemap document.

_PS: In parallel we are still giving some thoughts on whether "global" is a proper term/slug for those remaining events (include various W3C organized meetings such as Busdev events, Local Members meetings, Developer Meetups, Chapter events.)_

**Comment by vivienlacourba** [comment]
> Alex suggests we show "Events" in the menu and make all the other types children of it, available from an "Events" landing page.
> 
> Makes sense to me!

I agree with keeping only one "Events" entry as part of the "News & Events" menu.

But note that the [Events page](https://www-staging.w3.org/events/) is a listing of all events mixing the various types, and from here you can filter by event types.
The intro copy text for this listing (and each specific event listing text requested in https://github.com/w3c/w3c-website/issues/35) will I think provide enough context to the user.

The real landing page for that section is the to be written ["News & Events"  landing page](https://www-staging.w3.org/news-events).

> Where/how do I suggest this, please?

This change can be done directly in the CMS (under Structures / Main Navigation).

I have disabled the 4 extra event sub-types entries (namely "Talks, "Workshops", "Meetings"", "Conferences"), you can see the resulting menu on e.g. https://www-dev.w3.org/standards
@maries24, @koalie, does this chnage looks good to you?

**Comment by koalie** [comment]
It looks good to me, thanks!

**Comment by maries24** [comment]
I agree it looks fine. Please remember to replicate in the staging site :)

**Comment by koalie** [comment]
I have disabled each in place (it isn't obvious yet from https://cms-staging.w3.org/admin/entries/mainNavigation but it may need some time to propagate)

**Comment by koalie** [comment]
(i had not saved --now done and all good)

**Comment by JuliaSang** [comment]
@maries24 the outstanding action on this is to change the slug :

> the slug should be "happenings" instead of "global".

**Comment by maries24** [comment]
I updated the slug of meetings to 'happenings' on staging.
Preventing nesting of event types was addressed as part of another task earlier today.
